client/my-sites/stats/activity-log/index.jsx

```diff
@@ -591,9 +599,26 @@ class ActivityLog extends Component {
 					{ ...getActivityLogQuery( { gmtOffset, startDate, timezone } ) }
 				/>
 				<QuerySiteSettings siteId={ siteId } />
+				<QueryJetpackCredentials siteId={ siteId } />
 				<StatsFirstView />
 				<SidebarNavigation />
 				<StatsNavigation selectedItem={ 'activity' } siteId={ siteId } slug={ slug } />
+				{ ! hasMainCredentials && (
+					<CompactCard
+						highlight="info"
+						href={ credsUrl }
```

Comment:
This should be replaced with `'/settings/security/' + selectedSite.slug` since there's only one usage.